src/app/editor-container/editor-container.component.scss

```diff
@@ -0,0 +1,3 @@
+.editor-parent-container {
+  overflow-x: hidden;
```

Comment:
I am wondering if this might cause the preview of a PDF on the right side to be cut.

Try setting in the `app.config.ts` the [preview config](https://github.com/inveniosoftware-contrib/ng2-json-editor#previews) pointing to a `path` in the JSON document where there is a PDF. You will have to enable CORS on the browser to load the PDF from an external service.